src/main/java/org/cyclops/integrateddynamics/core/network/PartNetwork.java

```diff
@@ -127,6 +129,13 @@ public boolean hasPart(int partId) {
                     it.remove();
                 }
             }
+            // Also check parts
+            for(PartPos partPos : partPositions.valueCollection()){
```

Comment:
`for(PartPos partPos : partPositions.valueCollection()) {`

Reply:
Addressed below

Reply:
I think I addressed this below, or did you mean to say something else about this?

Reply:
Ah, finally spotted it -- there needed to be a space before the brace.
Fixed!